### ISSUE TITLE
Use Django 1.3/1.4 dictconfig logging configuration

### ISSUE THREAD

**Issue by jsocol** [mozilla/playdoh#108]
We should drop `settings/log_settings` (unless we really want to keep the whole dict in there) and, more importantly, the dependency on `nuggets/dictconfig` and just use Django's built-in [dictConfig-style logging configuration](https://docs.djangoproject.com/en/1.3/topics/logging/#an-example). This is less IH code to maintain, and should be more future proof and familiar to other Django devs.

We could keep `settings/log_settings` if we just wanted to do `from log_settings import LOGGING` in `settings.base` and have `LOGGING` be the normal `LOGGING` dict as defined in the docs.

tl:dr; If I'm not being clear, we should drop our custom stuff and configure Django logging as in the Django docs. It works in 1.3 and 1.4.
